e2e/cypress/integration/multi_team_and_dm/gm_add_first_user_spec.js

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+/* eslint-disable jquery/no-text */
+/* eslint-disable jquery/no-find */
+
+describe('Multi-user group messages', () => {
+    let testUser;
+    let testTeam;
+    before(() => {
+        // # Create a new team
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+
+            // # Add 3 users to the team that should be alphabetically sorted
+            ['0aadam', '0aabadam', 'beatrice'].forEach((prefix) => {
+                createTestUser(prefix, team);
+            });
+        });
+    });
+
+    it('MM-T459 Group Messaging: Add first user', () => {
+        const searchTerm = '0aa';
+
+        // # Login as test user
+        cy.apiLogin(testUser);
+
+        // # Go to town-square channel
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Open the 'Direct messages' dialog
+        cy.get('#addDirectChannel').
+            click();
+
+        // # Start typing part of a username that matches previously created users
+        cy.get('#selectItems input').
+            type(searchTerm);
+
+        // * Expect user list to only contain usernames matching the query term and to be sorted alphabetically
+        expectUserListSortedAlphabetically(searchTerm);
+
+        // # With the arrow and enter keys, select the first user that matches our search query
+        cy.get('body').
+            type('{downarrow}').
+            type('{enter}');
+
+        // * Expect to have the selected user's username to be displayed inside the search input
+        cy.get('.react-select__multi-value').
+            should('be.visible').
+            and('have.length', 1).
+            then(($selectedUser) => {
+                // # Get the selected user's username
+                const selectedUserName = $selectedUser.text();
+
+                // * Expect user list to not contain selected user, not be filtered, but still in alphabetical order
+                expectUserListSortedAlphabetically(selectedUserName, true);
+            });
+
+        // * Expect to have a remove button ('x') present next to the selected username (previous step)
+        cy.get('.react-select__multi-value__remove').
+            should('be.visible').
+            and('have.length', 1);
+
+        // * Expect search bar to have focus
+        cy.get('.react-select__multi-value').next().find('input').should('have.focus');
+
+        // * Expect previous search term to be cleared from search bar
+        cy.get('#selectItems input[type="text"]').should('have.text', '');
```

Comment:
Oh sorry I thought I had it. Here it is for consistency. Thanks for checking @Niceplace!
```suggestion
        cy.get('#selectItems input').should('have.text', '');
```